components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/models/Application.java

```diff
@@ -195,4 +195,13 @@ public HashMap getPermissionMap() {
     public void setPermissionMap(HashMap permissionMap) {
         this.permissionMap = permissionMap;
     }
+
+    @Override
+    public String toString() {
+        return "Application [name=" + name + ", groupId=" + groupId + ", uuid=" + uuid + ", policyId=" + policyId
```

Comment:
Based on this discussion http://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java , it doesn't seem to have any additional advantage for using stringbuilder